client/a8c-for-agencies/data/sites.ts

```diff
@@ -138,7 +142,9 @@ export const mockedSites: Site[] = [
 			mobile: 68,
 			overall: 82,
 		},
-		php_version_num: 7.4,
+		php_version_num: '7.4',
```

Comment:
The php_version_num field is a number returned by the endpoint

Reply:
Reverted back to number in fb88090.